pages/abilities.html

```diff
@@ -0,0 +1,195 @@
+                        </span>
+                    </li>
+                    <li>
+                        <span class='ability-title'>English</span>
+                        <span class='ability-score stars'>
+                            <span class="glyphicon glyphicon-star" aria-hidden="true"></span>
+                            <span class="glyphicon glyphicon-star" aria-hidden="true"></span>
+                            <span class="glyphicon glyphicon-star" aria-hidden="true"></span>
+                            <span class="glyphicon glyphicon-star" aria-hidden="true"></span>
+                            <span class="glyphicon glyphicon-star lite" aria-hidden="true"></span>
+                        </span>
+                    </li>
+                </ul>
+            </div>
+
+            
+            <div class="col-md-12 col-sm-12 col-xs-12 text-center" style='margin-bottom: 20px'>
+              <h2 class='head-one'>Technologies</h2>
+            </div>
+          
+          <div class="col-md-8 col-md-offset-2 col-sm-8 col-sm-offset-1 col-xs-12 text-center">
+            
+                    <div class="icon-list">
+                        <a target="_blank" href="https://developer.mozilla.org/en-US/docs/Glossary/HTML5" target="_blank">
+                        <img title="HTML" src="asset/img/html.png" alt="html">
+                        </a>
+                    </div>
+                    <div  class="icon-list">
+                        <a target="_blank" href="https://developer.mozilla.org/en-US/docs/Web/CSS/CSS3" target="_blank">
+                        <img title="CCS3" src="asset/img/css.png" alt="css">
+                        </a>
+                    </div>
+                    <div class="icon-list">
+                        <a target="_blank" href="http://getbootstrap.com/">
+                        <img title="Bootstarp" src="asset/img/bootstrap.png" alt="bootstrap" target="_blank">
+                        </a>
+                     </div>
+                     <div class="icon-list">
+                        <a target="_blank" href="http://jquery.com/">
+                        <img title="jQuery" src="asset/img/jquery.png" alt="jquery" target="_blank">
+                        </a>
+                     </div>
+                     <div class="icon-list">
+                        <a target="_blank" href="https://angularjs.org/">
+                        <img title="AngularJS" src="asset/img/angular.png" alt="angular" target="_blank">
+                        </a>
+                     </div>
+                     <div class="icon-list">
+                        <a target="_blank" href="https://www.firebase.com/">
+                        <img title="Firebase" src="asset/img/firebase.png" alt="firebase" target="_blank">
+                        </a>
+                    </div>
+
+                    <div class="icon-list">
```

Comment:
Each element here could be Angular component. Follow [DRY principle](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)